modules/apps/dynamic-data-mapping/dynamic-data-mapping-form-web/src/main/resources/META-INF/resources/admin/exceptions.jspf

```diff
@@ -103,13 +97,24 @@
 			<c:choose>
 				<c:when test='<%= msvv.getFieldName() == "expirationDate" %>'>
 					<liferay-ui:message key="please-enter-a-valid-expiration-date-only-future-dates-are-accepted" />
+					<% isShowingExpirationError = true; %>
 				</c:when>
 				<c:otherwise>
 					<liferay-ui:message arguments="<%= HtmlUtil.escape(fieldLabelValue) %>" key="validation-failed-for-field-x" translateArguments="<%= false %>" />
 				</c:otherwise>
 			</c:choose>
 		</liferay-ui:error>
 
+		<c:if test="<%= !isShowingExpirationError %>">
+			<c:if test="<%= ddmFormAdminDisplayContext.isFormExpired(ddmFormAdminDisplayContext.getDDMFormInstance()) %>">
```

Comment:
We can use just one `c:if`, if we combine the two clauses into one compound clause.
Like so:
`<c:if test="<%= !isShowingExpirationError && ddmFormAdminDisplayContext.isFormExpired(ddmFormAdminDisplayContext.getDDMFormInstance())%>">`

Reply:
Yes that's great, I'll do this!